lib/from_openstudio/model.rb

```diff
@@ -150,6 +153,20 @@ def self.materials_from_model(openstudio_model)
       openstudio_model.getBlinds.each do |material|
         result << EnergyWindowMaterialBlind.from_material(material)
       end
+      openstudio_model.getGass.each do |material|
```

Comment:
Interesting. Am I correct that this method won't return any `GasMixture` materials?

Reply:
Yes thats correct, Gas and GasMaterial are separate in OS.